test/common/testlib.py

```diff
@@ -122,11 +122,12 @@ def __init__(self, address, label, machine, pixels_label=None, port=None):
         self.pixels_label = pixels_label
         self.machine = machine
         path = os.path.dirname(__file__)
-        self.cdp = cdp.CDP("C.utf8", verbose=opts.trace, trace=opts.trace,
+        self.cdp = cdp.CDP("C.utf8", verbose=opts.trace, trace=opts.trace, fixed_content_size=self.pixels_label,
```

Comment:
So we indirectly rely on running pixel tests only on chromium? I think at least in cockpit-machines we run them on firefox also. But we can adjust that later on if it gets in the way.

Reply:
Well, the code only runs the hairy adjustment stuff when pixel tests are active.  If we activate pixel tests on Firefox, we will run into the "can't request specific window size" bug of Firefox and we will get the "can't adjust iframe" warning, but then the tests continue.

So if pixel tests are green on Firefox right now, they should remain green.  (No adjustement to the iframe have been made, and the window is still the same size because Firefox ignores --window-size.)